build/teamcity/cockroach/nightlies/microbenchmark_weekly.sh

```diff
@@ -84,5 +85,6 @@ fi
 # Generate sheets if comparing
 if [[ -n "${GCS_COMPARE_BINARIES}" ]] ; then
   ./bin/roachprod-microbench compare "$output_dir/0" "$output_dir/1" \
+    --slack-token "$MICROBENCH_SLACK_TOKEN" \
```

Comment:
Suggestion to make this match how the other parameter is passed to `roachprod-microbench`.

```suggestion
    --slack-token="$MICROBENCH_SLACK_TOKEN" \
```